app/tasks/email.py

```diff
@@ -60,13 +60,18 @@ def email_task(
     try:
         # 是否使用ssl
         if email_use_ssl:
-            client = smtplib.SMTP_SSL(email_smtp_host)
+            client = smtplib.SMTP_SSL(email_smtp_host, email_smtp_port)
```

Comment:
SMTP_SSL and SMTP constructors already do connect() inside. port must be given before that connect call.